server/room.ts

```diff
@@ -271,23 +84,11 @@ export class Room {
     };
   }
 
-  async resetRoomVM() {
-    const id = this.vBrowser && this.vBrowser.id;
-    this.vBrowser = undefined;
-    this.roster.forEach((user, i) => {
-      this.roster[i].isController = false;
-    });
-    this.cmdHost(undefined, '');
-    if (id) {
-      try {
-        await resetVM(id);
-      } catch (e) {
-        console.error(e);
-      }
+  cmdHost(socket: Socket, data: string | PlaylistVideo) {
```

Comment:
where did resetRoomVM go?

Reply:
moved it back to the room.ts, it's called stopVBrowser now to be more in line with startVBrowser